package/upgrade/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM registry.suse.com/suse/sle15:15.3
+
+ARG ARCH=amd64
+ENV ARCH=${ARCH}
+RUN zypper rm -y container-suseconnect && \
+    zypper ar --priority=200 http://download.opensuse.org/distribution/leap/15.3/repo/oss repo-oss && \
+    zypper --no-gpg-checks ref && \
+    zypper in -y curl e2fsprogs rsync awk zstd jq && zypper clean
+
+ENV KUBECTL_VERSION v1.20.4
```

Comment:
Is there any reason choosing 1.20.4 over newer version?

Reply:
Fixed. Use the same version of the current RKE2.